features/user_can_view_dashboard_page.feature

```diff
@@ -0,0 +1,39 @@
+Feature: User is sent to a dashboard where they can create or join a group.
+  As a user
+  In order to understand what functions are available to me as a new user
+  I would like to see a welcome page with directions of what to do next
+
+
+  Background:
+    Given the following user exists
+      |  email    | password    | password_confirmation |
+      | jf@ca.com | password123 | password123           |
+    And I am on the Landing page
+
```

Comment:
Please make use of the Warden helpers, the instructions are in the tips and tricks section.